bowling-ball/IFrames.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingBall
+{
+    public interface IFrames

```

Comment:
can you explain why you are using interface?

Reply:
I am using Moq in the code. But as Travis was giving errors as moq nuget was not available in the yml file, so I commented that part in the code. In the Test Cases , it is till there (commented).